testing/simapp/ante.go

```diff
@@ -1,37 +1,43 @@
 package simapp
 
 import (
-	errorsmod "cosmossdk.io/errors"
+	"errors"
+
+	circuitante "cosmossdk.io/x/circuit/ante"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/auth/ante"
 
 	ibcante "github.com/cosmos/ibc-go/v7/modules/core/ante"
-	ibcerrors "github.com/cosmos/ibc-go/v7/modules/core/errors"
 	"github.com/cosmos/ibc-go/v7/modules/core/keeper"
 )
 
-// HandlerOptions extend the SDK's AnteHandler options by requiring the IBC keeper.
+// HandlerOptions are the options required for constructing a default SDK AnteHandler.
 type HandlerOptions struct {
 	ante.HandlerOptions
-
-	IBCKeeper *keeper.Keeper
+	CircuitKeeper circuitante.CircuitBreaker
```

Comment:
do we need this circuit breaker? what is it used for and if its important why isn't it included in the embedded `ante.HandlerOptions`?